modules/ROOT/pages/amazon/amazon-sqs-connector-xml-maven.adoc

```diff
@@ -0,0 +1,72 @@
+= Amazon SQS XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+If you manually code a Mule app in XML either from the Anypoint Studio XML editor or from a text editor, you can get access to Amazon SQS Connector in your app by adding reference statements to your XML Mule flow and in the Apache Maven `pom.xml` file.
+
+[NOTE]
+====
+You can avoid the manual process described in this topic by using Studio. When you install Amazon SQS Connector in Studio, Studio automatically populates the XML code with the Amazon SQS Connector namespace and schema location, and adds a dependency to the `pom.xml` file.
```

Comment:
```suggestion
You can avoid the manual process described next by using Studio. When you install Amazon SQS Connector in Studio, Studio automatically populates the XML code with the Amazon SQS Connector namespace and schema location, and adds a dependency to the `pom.xml` file.
```
Do we have anything to add about WHY a user might PREFER this manual process? Why are we documenting it?